core/src/main/scala/org/bitcoins/core/p2p/NetworkPayload.scala

```diff
@@ -1009,6 +1009,20 @@ case class CompactFilterHeadersMessage(
       acc :+ acc.last.nextHeader(nextFilterHash)
     }
   }
+
+  override def toString: String = {
+    val hashesString = {
+      if (filterHashes.isEmpty) {
+        "empty"
+      } else {
+        s"${filterHashes.head}...${filterHashes.last}"
+      }
+    }
+    s"CompactFilterHeadersMessage(filterType=${filterType}, " +
+      s"previousFilterHeader=${previousFilterHeader}, " +
+      s"stopHash=${stopHash} " +
```

Comment:
can you make this `.flip.hex` that way we get the `DoubleSHA256DigestBE` version and then just the `hex` string for it